### PR TITLE
SEO update for how-to-roll-out-teams

### PR DESCRIPTION
**Pull request by jdmartinez36**
Updated description to add keywords, added keywords to H2 headings, replaced some instances of "rollout" with "implementation" per the recommendations.

### CODE REVIEW

**Comment by jdmartinez36** [comment]
@DaniHalfin

**Comment by DaniHalfin** [comment]
Assigned to myself - no need to review.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/How-to-roll-out-teams.md [hunk 1]
```diff
@@ -38,21 +39,21 @@ We recommend rolling out Teams in stages, workload by workload, as your organiza
...
+[Adoption hub](adopt-microsoft-teams-landing-page.md): Throughout your Teams implementation, be sure to take advantage of these resources to help drive Teams adoption.
 
 ![Diagram illustrating Teams deployment pathways](media/how-to-roll-out-teams-image1.png)
 
 
 ## If you're starting from Skype for Business, on-premises, or hybrid deployments
 
-If you're coming to Teams from Skype for Business (online or on premises), or if you need a hybrid configuration, you still want to follow the [recommended path](#recommended-path-to-teams) above for rolling out Teams, but first you need to do some extra planning. Start by reviewing the guidance in the table below that applies to your organization's profile.
+If you're coming to Teams from Skype for Business (online or on premises), or if you need a hybrid configuration, you still want to follow the [recommended path](#recommended-path-to-teams) above for a Teams deployment, but first you need to do some extra planning. Start by reviewing the guidance in the table below that applies to your organization's profile.
```
Comment (JohanFreelancer9) [inline Teams/How-to-roll-out-teams.md:49]:
```suggestion
If you're coming to Teams from Skype for Business (online or on-premises), or if you need a hybrid configuration, you still want to follow the [recommended path](#recommended-path-to-teams) above for a Teams deployment, but first you need to do some extra planning. Start by reviewing the guidance in the table below that applies to your organization's profile.
```

**Comment by scanum** [comment]
@DaniHalfin Hi! Is this PR going to be closed? Thanks!

**Pull request merged**